### DISCUSSION TITLE
Question about Evaluation metric vs. Loss

### DISCUSSION THREAD

**Discussion by mjbooo** [seominjoon/kaist-ai605#70] [Q&A] [answered]
In the 'Evaluation Metroc vs Loss(2)', it said ' Another viable strategy is to use a low-variance with a reasonably low bias in the initial stage, and switch to an unbiased yet high variance loss in the later stage'.
But why is it reasonable to use a low bias in the initial stage, not the other way around? 

Thank you

**Comment by seominjoon** [comment, marked as answer]
I see where you got confused. What I wanted to say is that lowering the variance at the cost of increasing the bias initially would be a viable strategy. But even then, a loss that has very high bias is basically a useless loss, so it will need to be sufficiently low, which is why i said "low bias".
